File: PHP/ReplaceWithAlphabetPosition/MySolution.php
<?php

function alphabet_position(string $s): string {
  
  $numberLetters = array_combine(range('a', 'z'), range(1, 26));
  $s = strtolower($s);
  $result = [];

  foreach (str_split($s) as $letter) {
      if (isset($numberLetters[$letter])) {
          $result[] = $numberLetters[$letter];
      }
  }

  return implode(" ", $result);
}

// Original kata: https://www.codewars.com/kata/546f922b54af40e1e90001da
// My solution: https://www.codewars.com/kata/reviews/59b3c8bfd0ca2d061400014d/groups/67bd7139f73ac331ad64fc43

File: Python/NestringStructure/MySolution.py
def same_structure_as(original,other):
    if isinstance(original, list) and isinstance(other, list):
        if len(original) != len(other):
            return False
        return all(same_structure_as(o1, o2) for o1, o2 in zip(original, other))
    return not isinstance(original, list) and not isinstance(other, list)

# Original kata: https://www.codewars.com/kata/520446778469526ec0000001
# My solution: https://www.codewars.com/kata/reviews/554662240240a7b6f800019c/groups/664f660df987f1ab5fc53e16
